src/App/Tracker/Controller/Issue/Save.php

```diff
@@ -333,4 +339,133 @@ private function updateGitHub($issueNumber, array $data, $state, $oldState)
 
 		return $this;
 	}
+
+	/**
+	 * Add the status as label on GitHub.
+	 *
+	 * The method gets the list of current labels and remove all labels that starts with `~`
+	 * as it is the status prefix, after this the method add the new status label
+	 *
+	 * @param   integer  $issueNumber  The issue number.
+	 * @param   string   $newlabel     The new label that should be added as label.
+	 *
+	 * @throws \Exception
+	 * @throws \JTracker\Github\Exception\GithubException
+	 *
+	 *
+	 * @return  $this
+	 *
+	 * @since   1.0
+	 */
+	private function addLabelToGitHub($issueNumber, $newlabel)
+	{
+		/* @type \JTracker\Application $application */
+		$application = $this->getContainer()->get('app');
+
+		// Get the labels for the pull's issue
+		try
+		{
+			$labels = $github->issues->get($project->gh_user, $project->gh_project, $issueNumber)->labels;
+		}
```

Comment:
thanks fixed with https://github.com/zero-24/jissues/commit/112126e43e33e72c0acc93f378a62b77568a172e als the `$project` was undefined here